src/utils/utils.py

```diff
@@ -20,7 +20,7 @@
 from torch.nn import Parameter
 from torch.nn import init
 
-from allennlp.nn.util import masked_log_softmax
+from allennlp.nn.util import masked_softmax
```

Comment:
why do we not need to take log softmax anymore? Taking log softmax is usually more numerically stable